src/LimitCalib.cpp

```diff
@@ -0,0 +1,6 @@
+#include "world_interface/world_interface.h"
+
+int main() {
+	bool runCalibration = true;
+	robot::world_interface_init(runCalibration);
```

Comment:
As we discussed in the discord this should add a limit switch callback, run the motors until it hits a limit switch, and then in the callback stop the motor power and exit.